### ISSUE TITLE
VDPAU: Failed to open VDPAU backend libvdpau_amdgpu.so

### ISSUE THREAD

**Issue by hosaka** [corngood/archlinux-amdgpu#9]
Trying to get some info about VDPAU yields:
```
% vdpauinfo
display: :0   screen: 0
Failed to open VDPAU backend libvdpau_amdgpu.so: cannot open shared object file: No such file or directory
Error creating VDPAU device: 1
```
This might be due to the fact that the installed lib is located at `/usr/lib/x86_64-linux-gnu/vdpau/libvdpau_amdgpu.so`, symlinking it to `/usr/lib` solves the problem:
```
API version: 1
Information string: G3DVL VDPAU Driver Shared Library version 1.0
```

**Comment by corngood** [comment]
I'm in the process of moving all those libs into /usr/lib/ and /usr/lib32/ instead of the debian style paths.  I'll make sure to test vdpau when I'm done.

Thanks for the report.

**Comment by hosaka** [comment]
Awesome, thank you! Closed.

**Comment by corngood** [comment]
It's still a legit bug, so I'll leave it open until it's fixed.

**Comment by corngood** [comment]
Fixed By #10
